src/researchhub_document/views/researchhub_unified_document_views.py

```diff
@@ -81,64 +82,48 @@ def get_queryset(self):
             return ResearchhubUnifiedDocument.all_objects.all()
         return super().get_queryset()
 
-    @action(
-        detail=False,
-        methods=["get"],
-        permission_classes=[AllowAny],
-    )
+    @action(detail=False, methods=["get"], permission_classes=[AllowAny])
     def recommendations(self, request, *args, **kwargs):
-        user_id = request.query_params.get("user_id", None)
+        user_id = request.query_params.get("user_id")
+        if not user_id:
+            return Response({"error": "user_id is required"}, status=400)
+
         personalize_runtime = boto3.client(
             "personalize-runtime", region_name="us-west-2"
         )
-
         campaign_arn = "arn:aws:personalize:us-west-2:794128250202:campaign/hp-recs"
```

Comment:
This hard-coded value is temporary and safe to expose. This is a shim until we write a new API for mixed content in next sprint.